src/init.ts

```diff
@@ -26,6 +27,12 @@ export class Init {
         return (await readPkg(path.join(this.rootFolder, 'packages/core/package.json'))).version;
     }
 
+    async getPackageWithVersion(name: string): Promise<String> {
+        const command = new Command(path.resolve('.'));
+        const fullPkg = await command.exec('yarn --json --non-interactive --no-progress list --pattern=' + name + " | jq --raw-output '.data.trees[0].name'");
```

Comment:
we've a yarn class so maybe delegate yarn stuff to this class ?

Reply:
Was not really sure how to use it for this purpose TBH

Reply:
BTW we should not use jq there. just handle result in json object which is native in JavaScript

Reply:
Oh, sure.